.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+# env:
+#   JULIA_NUM_THREADS: 2
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+    tags: '*'
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0' # Replace this with the minimum Julia version that your package supports.
+          - '1'   # automatically expands to the latest stable 1.x release of Julia
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+        # include:
+        #   - os: windows-latest
+        #     version: '1'
+        #     arch: x64
+        #   - os: macOS-latest
+        #     version: '1'
+        #     arch: x64
```

Comment:
```suggestion
        include:
          - os: windows-latest
            version: '1'
            arch: x64
          - os: macOS-latest
            version: '1'
            arch: x64
```

Reply:
BSON interacts with system deps, the tests aren't very costly, so good to test for different LOSs